src/wallet/api/wallet.cpp

```diff
@@ -1429,75 +1428,78 @@ PendingTransaction *WalletImpl::createTransaction(const string &dst_addr, const
     PendingTransactionImpl * transaction = new PendingTransactionImpl(*this);
 
     do {
-        if(!cryptonote::get_account_address_from_str(info, m_wallet->nettype(), dst_addr)) {
-            // TODO: copy-paste 'if treating as an address fails, try as url' from simplewallet.cpp:1982
-            setStatusError(tr("Invalid destination address"));
+        bool payment_id_seen = false;
```

Comment:
Can drop `payment_id_seen` and use `!extra_nonce.empty()` instead. Though I'm fine with both ways, let me know if you would like to keep it as-is.

Reply:
updated it to !extra_nonce.empty(), saving some diff lines :D